src/main/java/core/CoreConstants.java

```diff
@@ -57,9 +57,12 @@ public enum ComponentType {
     }
 
     public enum GameResult {
-        WIN(1),
-        DRAW(0),
-        LOSE(-1),
+        WIN_GAME(1),
```

Comment:
Not sure GameResult is the right place to store WIN_ROUND statuses - but given LoveLetter is doing this anyway this is still an improvement!
However, I do think the values for any ROUND status should be 0.0, as the game is ongoing.